### ISSUE TITLE
Exception at ' 2 * =>-<= 0 - ' in section [TOP] (- can only operate on numeric values, vectors, matrices and numeric Geo Time Series.)

### ISSUE THREAD

**Issue by alexandrevilain** [ovh/erlenmeyer#47]
Hello 🙂 

I’m facing an issue on a PromQL query on metrics' grafana which I don’t have on a « raw » prometheus instance. 

Here is my request on grafana: `1 - avg(rate(node_cpu_seconds_total{mode="idle", cluster="$cluster"}[$__interval]))`

Here is the response on the « raw » prometheus instance: 
```
{"status":"success","data":{"resultType":"vector","result":[{"metric":{},"value":[1604911311,"0.16644146550188865"]}]}}
```

Here is the response on metrics:
```
{"status":"error","data":{"resultType":"","result":null},"errorType":"execution","error":"Exception at ' 2 * =\u003e-\u003c= 0 - ' in section [TOP] (- can only operate on numeric values, vectors, matrices and numeric Geo Time Series.)"}
```

FYI, the data is pushed on metrics thanks to prometheus' [remote write](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) feature.

I'm unable to find where is the issue, but I'll be happy to help you if needed. :)

**Comment by alexandrevilain** [comment]
It's an instant query, here is the grafana query url:

```
api/datasources/proxy/1/api/v1/query?query=1%20-%20avg(rate(node_cpu_seconds_total%7Bmode%3D%22idle%22%2C%20cluster%3D%22%22%7D%5B120s%5D))&time=1604913319
```

**Comment by aurrelhebert** [comment]
Hello @alexandrevilain

**Comment by aurrelhebert** [comment]
Should be fixed by #48

**Comment by alexandrevilain** [comment]
Awesome! Thanks @aurrelhebert ! :)
